### PR TITLE
Adding network table doc

### PR DESCRIPTION
**Pull request by SgtCoin**
Adding network update table

### CODE REVIEW

**Comment by SgtCoin** [comment]
I changed the name to README.md so it will render correctly and added the missing fips into the v13 and v14 sections.

File: Network Upgrades/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+| network upgrade # | upgrade name | actors version |                     FIPs included                    |  Epoch  |
```
Comment (arajasek) [inline Network Upgrades/README.md:1]:
```suggestion
| Upgrade # | Upgrade name | Actors Version |                     FIPs included                    |  Epoch  |

```

File: Network Upgrades/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+|:-----------------:|:------------:|:--------------:|:----------------------------------------------------:|:-------:|
+|        v06        |    Kumquat   |     v2.2.0     |                       FIP-0004                       |  170000 |
+|        v07        |    Calico    |     v2.3.2     |                       FIP-0005                       |  265200 |
+|        v08        |    Persian   |     v2.3.2     |                        No FIPs                       |  272400 |
+|        v09        |    Orange    |     v2.3.3     |                        No FIPs                       |  336458 |
+|       v09.5       |     Claus    |     v2.3.3     |                       FIP-0009                       |  343200 |
+|        v10        |     Trust    |     v3.0.3     |                       FIP-0010                       |  550321 |
+|        v11        |   Norwegian  |     v3.1.0     |                       FIP-00014                      |  665280 |
+|        v12        |     Turbo    |     v4.0.0     |                        No FIPs                       |  712.20 |
```
Comment (arajasek) [inline Network Upgrades/README.md:10]: epoch seems wrong?

File: Network Upgrades/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+|:-----------------:|:------------:|:--------------:|:----------------------------------------------------:|:-------:|
+|        v06        |    Kumquat   |     v2.2.0     |                       FIP-0004                       |  170000 |
+|        v07        |    Calico    |     v2.3.2     |                       FIP-0005                       |  265200 |
+|        v08        |    Persian   |     v2.3.2     |                        No FIPs                       |  272400 |
+|        v09        |    Orange    |     v2.3.3     |                        No FIPs                       |  336458 |
+|       v09.5       |     Claus    |     v2.3.3     |                       FIP-0009                       |  343200 |
+|        v10        |     Trust    |     v3.0.3     |                       FIP-0010                       |  550321 |
+|        v11        |   Norwegian  |     v3.1.0     |                       FIP-00014                      |  665280 |
+|        v12        |     Turbo    |     v4.0.0     |                        No FIPs                       |  712.20 |
+|        v13        |  HyperDrive  |     v5.0.0     | FIP-0008 / FIP-0011 / FIP-0012 / FIP-0013 / FIP-0015 |  892800 |
```
Comment (arajasek) [inline Network Upgrades/README.md:11]: Nit: Maybe separate the FIPs by commas?

**Review by arajasek — CHANGES_REQUESTED** [review]
Thanks for the PR! Just a couple nits.

File: Network Upgrades/README.md (hunk 2 above)
Comment (SgtCoin) [reply Network Upgrades/README.md:10]: I see it... typo.. fixing

File: Network Upgrades/README.md (hunk 3 above)
Comment (SgtCoin) [reply Network Upgrades/README.md:11]: will do

**Review by SgtCoin — COMMENTED** [review]
Fixed all the things.

**Review by arajasek — APPROVED** [review]
Looks good to me!

It would be nice to add links to the FIPs and actors tags (see [here](https://github.com/filecoin-project/tpm/blob/master/Network%20Upgrades/v11.md) for an example), but it's not necessary.

File: Network Upgrades/README.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+|        v07        |    Calico    |     v2.3.2     |                       [FIP-0005](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0005.md)                       |  265200 |
+|        v08        |    Persian   |     v2.3.2     |                        No FIPs                       |  272400 |
+|        v09        |    Orange    |     v2.3.3     |                        No FIPs                       |  336458 |
+|       v09.5       |     Claus    |     v2.3.3     |                       [FIP-0009](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0009.md)                       |  343200 |
+|        v10        |     Trust    |     v3.0.3     |                       [FIP-0010](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0010.md)                       |  550321 |
+|        v11        |   Norwegian  |     v3.1.0     |                       [FIP-00014](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0014.md)                      |  665280 |
```
Comment (arajasek) [inline Network Upgrades/README.md:9]:
```suggestion
|        v11        |   Norwegian  |     v3.1.0     |                       [FIP-0014](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0014.md)                      |  665280 |

```

File: Network Upgrades/README.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+| Upgrade #         | Upgrade name | Actors Version |                     FIPs included                    |  Epoch  |
```
Comment (arajasek) [inline Network Upgrades/README.md:1]: Can we add a 

## Network Upgrades to the start of this file

**Review by arajasek — APPROVED** [review]
2 last nits

File: Network Upgrades/README.md (hunk 4 above)
Comment (SgtCoin) [reply Network Upgrades/README.md]: Fixed

File: Network Upgrades/README.md (hunk 5 above)
Comment (SgtCoin) [reply Network Upgrades/README.md:3]: Added

**Comment by kaitlin-beegle** [comment]
This is a great tool! Thanks for organizing this, @SgtCoin! 

@arajasek I wonder if it's worth adding an explainer, re: FIPs 2, 6, and 7? I assume these were implemented as need-to-have protocol enhancements outside of a standard upgrade, but it might be nice to add clarifying details as to why.

**Comment by arajasek** [comment]
@kaitlin-beegle Taking them one-by-one:

- FIP-0002 was adopted in network version which was before mainnet -- the NV docs I added in #78 only cover the mainnet-and-after versions. I could add those docs too, though, if you'd like.
- FIP-0006 was deferred, it never went live.
- FIP-0007 went live in nv10, I totally missed that in my PR. Gonna add it there, and @SgtCoin can you please add it here too?

**Pull request merged**